onnxruntime/core/providers/cpu/signal/dft.cc

```diff
@@ -273,13 +265,15 @@ static Status dft_naive(const Tensor* X, Tensor* Y,
   auto angular_velocity = compute_angular_velocity<T>(dft_length, inverse);
 
   for (size_t i = 0; i < dft_output_size; i++) {
-    std::complex<T>& out = *(Y_data + i*Y_stride);
+    std::complex<T>& out = *(Y_data + i * Y_stride);
     out.real(0);
     out.imag(0);
 
     for (size_t j = 0; j < dft_length; j++) {  // vectorize over this loop
-      auto exponential = std::complex<T>(cos(i * j * angular_velocity), sin(i * j * angular_velocity));
-      auto window_element = window_data ? * (window_data + j) : 1;
+      auto exponential = std::complex<T>(
+          cos(static_cast<T>(i) * static_cast<T>(j) * angular_velocity),
```

Comment:
I don't know if the compiler is able to detect the same expression is used just below.